ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -55,4 +55,29 @@ - (void)testShouldIgnoreVnodeType {
     XCTAssertFalse(ShouldIgnoreVnodeType(static_cast<vtype>(1000), testVnode.get()));
 }
 
+- (void)testFileFlaggedInRoot {
+    bool fileFlaggedInRoot;
+    MountPointer testMount = mount::Create("hfs", fsid_t{}, 0);
```

Comment:
We should probably set sane default values for `Create()` to avoid cluttering our tests with specific values when we don't actually care.